packages/mira/tasks/sbml_to_petrinet.py

```diff
@@ -4,13 +4,29 @@
 from mira.metamodel.ops import simplify_rate_laws
 from mira.modeling import Model
 from mira.modeling.amr.petrinet import AMRPetriNetModel
-from mira.sources.sbml import template_model_from_sbml_string
+from mira.sources.sbml import template_model_from_sbml_file
 
 
 def cleanup():
     pass
 
 
+def get_filename(id: str):
+    return f"/tmp/{id}.stella"
+
+
+def create_temp_file(name, contents):
+    with open(name, 'w') as f:
+        f.write(contents)
+
+
+def delete_temp_file(name):
+    try:
+        os.remove(name)
+    except:
+        pass
```

Comment:
Should this be logged?

Reply:
Nah, its just to clean up any created file, if it never made it far enough to create, this just catches that "file does not exist"  error.